DavidVigil/frontend-project
# Apps Main/Landing Page

## Description:

### This pull request introduces several enhancements and features to improve the user experience and functionality of the application:

- Background Animation and Layout Improvement:

  - Added an animated background GIF to enhance the visual appeal of the application.

  - The main content is now displayed within a transparent container with a blur effect to maintain readability against the dynamic background.

- Search and Filter Capabilities:

  - Implemented a search bar that filters the list of applications based on the user's input.

  - Added a filter button with a popup dialog allowing users to filter applications by their source ("FI" or "Extern") or view all applications.

- User Interaction Enhancements:

  - Incorporated a "Save" button that opens a dialog prompting users to sign in if they haven't already.

  - Enhanced the responsiveness and layout by aligning the search bar and filter button, ensuring a user-friendly interface.

## To Test:

- Run the application and verify the following:
  - The background animation displays correctly, and the main content is readable.
  - The search bar filters applications as expected.
  - The filter button opens a popup with filtering options.
  - The save button prompts for sign-in if the user is not logged in.
